.github/workflows/docker_promote.yml

```diff
@@ -19,10 +19,10 @@ on:
   workflow_dispatch:
     inputs:
       rc_docker_image:
-        description: RC docker image that needs to be promoted (Example:- apache/kafka:3.6.0-rc0)
+        description: RC docker image that needs to be promoted (Example:- apache/kafka:3.6.0-rc0, apache/kafka-native:3.6.0-rc0)
```

Comment:
nit: it can be a bit confusing as the input is a text box and in example we have provide comma separated values. Although the title says image, so it should be okay, that only single input is expected.

Reply:
maybe use `apache/kafka:3.6.0-rc0`  (or) `apache/kafka-native:3.6.0-rc0`

Reply:
also can we use `3.7.0` version in all the examples

Reply:
> maybe use apache/kafka:3.6.0-rc0 (or) apache/kafka-native:3.6.0-rc0
+1

native won't have 3.7.0 tag. Should we consider making it 3.8.0 in all examples?

Reply:
Updated all the example messages with `3.8.0` version and included `OR`.